lib/deploy/deploy.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+
+module.exports = {
+  /*
+   * Will get stack resources and create object and append in obj
+   * with key as ARN Name and value as ARN Value
```

Comment:
Please add the description that you explained to me.

Reply:
Added detailed description